Basics/Writing bug reports.md

```diff
@@ -11,110 +11,151 @@ Each bug report, at a minimum, consists of a title and a description.
 ## Title
 
 The title can either be:
-
 - An imperative denoting what needs to be done.
 	- E.g. "Add an Info button to the login screen"
 - Or a description of the error.
 	- E.g. "The app crashes when I take a photo of myself"
 
-## Description
```

Comment:
Do we maybe want to keep "Description" subtitle? As it seems it is a border between the title and the rest of the bug report?

Reply:
Updated in new commit